### PR TITLE
Rename module directories to match Maven artifact names

### PR DESCRIPTION
**Pull request by twyatt**
Renames module directories to align with Gradle Maven conventions for artifact naming (rather than leaning on POM publication plugin).

This allows Kable to play nicer when pulled in as a Gradle [composite build](https://docs.gradle.org/current/userguide/composite_builds.html) (e.g. https://github.com/JuulLabs/sensortag/pull/363).

### CODE REVIEW

**Review by cedrickcooke — APPROVED** [review]

**Review by sdonn3 — APPROVED** [review]

**Comment by twyatt** [comment]
I'll wait until #706 is merged to merge this PR.

**Pull request merged**
